api/src/opentrons/protocol_engine/commands/__init__.py

```diff
@@ -341,9 +343,11 @@
     "LoadLiquidParams",
     "LoadLiquidResult",
     "LoadLiquidCommandType",
-    # module command bundles
+    # hardware module command bundles
     "heater_shaker",
     "magnetic_module",
     "temperature_module",
     "thermocycler",
+    # calibration module command bundles
```

Comment:
```suggestion
    # calibration command bundle
```

Since there's only one bundle of calibration commands.